src/main/telemetry/telemetry.c

```diff
@@ -76,7 +82,7 @@ PG_RESET_TEMPLATE(telemetryConfig_t, telemetryConfig,
             IBUS_SENSOR_TYPE_RPM_FLYSKY,
             IBUS_SENSOR_TYPE_EXTERNAL_VOLTAGE
     },
-    .smartport_use_extra_sensors = false,
+    .disabledSensors = ALL_ESC_SENSORS,
```

Comment:
May be better to use `.enabledSensors`, so sensors are disabled by default (zero config value) and newly added sensors are not enabled silently